extensions/mssql/src/features.ts

```diff
@@ -1183,3 +1183,50 @@ export class TableDesignerFeature extends SqlOpsFeature<undefined> {
 	}
 }
 
+
+/**
+ * Execution Plan Service Feature
+ * TODO: Move this feature to data protocol client repo once stablized
+ */
+export class ExecutionPlanServiceFeature extends SqlOpsFeature<undefined> {
+	private static readonly messagesTypes: RPCMessageType[] = [
+		contracts.GetExecutionPlanRequest.type,
+	];
+
+	constructor(client: SqlOpsDataClient) {
+		super(client, ExecutionPlanServiceFeature.messagesTypes);
+	}
+
+	public fillClientCapabilities(capabilities: ClientCapabilities): void {
+	}
```

Comment:
Nitpick: Leave a comment saying this method definition is intentionally being left empty, so that no one in the future thinks it was something we forgot to define.